### PR TITLE
docs: timezone setting

### PR DESCRIPTION
**Pull request by Taylor-lagrange**
## What's Changed in this PR

Docs about how to use set timezone under different protocol, and what influence time zone will bring in query and insert

https://github.com/GreptimeTeam/greptimedb/issues/2907

## Checklist

- [x] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

**Review by nicecui — APPROVED** [review]

File: docs/v0.7/en/user-guide/clients/http-api.md [hunk 1]
```diff
@@ -49,11 +49,55 @@ http://localhost:4000/v1/sql?db=public
 InfluxDB uses its own authentication format, see [InfluxDB](./influxdb-line.md) for details.
 :::
 
-## Write Data
+## Time zone
+
+GreptimeDB supports the `X-Greptime-Timezone` field in the header of the HTTP protocol.
```
Comment (tisonkun) [inline docs/v0.7/en/user-guide/clients/http-api.md:54]:
```suggestion
GreptimeDB supports the `X-Greptime-Timezone` header in HTTP requests.
```

IIRC HTTP Xxx header is a formal description. cc @nicecui  - https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type "a Content-Type header"

File: docs/v0.7/en/user-guide/clients/http-api.md [hunk 2]
```diff
@@ -49,11 +49,55 @@ http://localhost:4000/v1/sql?db=public
...
+It is used to specify the timezone for the current SQL query.
+
+For example, the following request uses the time zone `+1:00` for the query:
```
Comment (tisonkun) [inline docs/v0.7/en/user-guide/clients/http-api.md:57]:
```suggestion
For example, the following request uses the timezone `+1:00` for the query:
```

If we use `timezone` consistently, let's keep it.

**Comment by killme2008** [comment]
@tisonkun PTAL

**Comment by tisonkun** [comment]
Seems "time zone" the proper expression. I'll update.

**Review by tisonkun — APPROVED** [review]

**Pull request merged**